### ISSUE TITLE
Add release notes for new versions

### ISSUE THREAD

**Issue by joshbainbridge** [shell-pool/shpool#165]
I've been following along with shpool for a while, it's an interesting project! I also use it here and there. My question would be, are there release notes for each new version that would help me understand what has changed?

If not, perhaps keeping something like a [CHANGELOG.md](https://keepachangelog.com/en/1.0.0/) file might provide value to others that follow along, but aren't keeping track of the changes day-to-day.

**Comment by ethanpailes** [comment]
The changelog is here: https://github.com/shell-pool/shpool/blob/master/debian/changelog

It is in debian format because at one point we built shpool .deb files using some mostly standard debian build tooling as part of the internal google release process. We've since moved on from that, but keep the debian tree around to make life easier for anyone who wants to package it (which we might try to do ourselves at some point, though don't have any immediate plans to do so).

We probably should link to this from the main README though.
